### DISCUSSION TITLE
How to rerun outliners while using parameters?

### DISCUSSION THREAD

**Discussion by gavrilikhin-d** [sharkdp/hyperfine#748] [Q&A]
I run the following benchmark:
```bash
 hyperfine -w 1 -r 5 --export-csv ppl.csv  -P n 1 100 "N={n} ./target/linalg.out"
```
Some of the benchmarks have outliners. Is it possible to rerun only those benchmarks? 

I don't want to increase number of warmups, because then benchmark would take a lot of time and it doesn't even help

**Comment by sharkdp** [comment]
This is currently not possible.
